domains/Megalith.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "MegalithOffical",
+        "email": "gekocakaya@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "https://profile-megalithoffical.vercel.app/"
```

Comment:
Sorry, we don't support Vercel, but you can change the record type to `URL` and have it redirect to the vercel app.